src/ExternalLoaderPlugin.ts

```diff
@@ -0,0 +1,115 @@
+import Compiler = require('webpack/lib/Compiler');
```

Comment:
do you think we should put this in its in folder too rather than just `src`? `external-loader-plugin` maybe?

Reply:
Yeah, makes sense. I put it in a folder and added an `index.ts` so it can still be imported as `@dojo/webpack-contrib/external-loader-plugin` instead of `@dojo/webpack-contrib/external-loader-plugin/ExternalLoaderPlugin`.